checks/remotesettings/certificates_expiration.py

```diff
@@ -50,23 +65,38 @@ async def run(server: str, min_remaining_days: int) -> CheckResult:
 
     # Second, deduplicate the list of x5u URLs and fetch them in parallel.
     x5us = list(set(metadata["signature"]["x5u"] for metadata in results))
-    futures = [fetch_cert(x5u) for x5u in x5us]
+    futures = [fetch_certs(x5u) for x5u in x5us]
     results = await run_parallel(*futures)
-    expirations = {
-        x5u: cert.not_valid_after.replace(tzinfo=datetime.timezone.utc)
-        for x5u, cert in zip(x5us, results)
-    }
+
+    validity: Dict[str, Tuple] = {}
+    for x5u, certs in zip(x5us, results):
+        # For each cert of the chain, keep track of the one that ends the earliest.
```

Comment:
This is just recording all of the ending times, not finding the one that ends the earliest, right? Is this comment out of date?

Reply:
No, we have ` or end < validity[x5u][0]` below

Reply:
Ah, I understand now. :+1: